### PR TITLE
made multimodule project, added missing relationships to entities

### PR DESCRIPTION
**Pull request by msmidik**
dir fleet-management renamed to fleet-management-persistence... not sure if it break something

### CODE REVIEW

**Comment by Qerts** [comment]
@msmidik  jop, you should adjust .yaml for CI, there is project path used in the script

**Comment by msmidik** [comment]
tak travisovi se to prejmenovani nelibi...

**Comment by trebi** [comment]
@msmidik I see that you split project into two sub-directories, OK. But please keep `.idea` in root folder and copy `.gitignore` to subfolders. Also, you could make the sub-directories the modules of the project ([Project (IDEA) == Workspace (Eclipse) and Module (IDEA) == Project (Eclipse)](http://superuser.com/questions/850373/does-intellij-idea-have-the-concept-of-workspace-similar-to-eclipse-with-multi))

One you will have correctly set up the `.gitignore`, remove files that would be normally ignored (e.g. in `.idea/libraries`). 

Then you need of course update the `.travis.ci` file, so build will pass.

If you will have some issues with this, write it to slack of wait for the tomorrow seminar - we will solve it there.

**Comment by trebi** [comment]
@msmidik Thanks, but there is still `.idea` folder in `fleet-management-persistence` and you are still commiting 98 files, where most of them are completely unnecessary (like everything in folder ` .idea/libraries/`). Please remove them - they do not belong to the repository and code review will be easier for us.

File: fleet-management-persistence/src/main/java/cz/fi/muni/pa165/entity/Employee.java [hunk 1]
```diff
@@ -55,6 +64,22 @@ public void setSurname(String surname) {
         this.surname = surname;
     }
 
+    public Set<VehicleCategory> getVehicleCategories() {
+        return vehicleCategories;
+    }
+
+    public void setVehicleCategories(Set<VehicleCategory> vehicleCategories) {
+        this.vehicleCategories = vehicleCategories;
```
Comment (trebi) [inline fleet-management-persistence/src/main/java/cz/fi/muni/pa165/entity/Employee.java]: Write test, if you set new `Categories` to `Employee`, if also the `employee` collection of each `Category` gets updated.

File: fleet-management-service/src/main/java/cz/fi/muni/pa165/service/VehicleService.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import cz.fi.muni.pa165.entity.Vehicle;
+
+import java.util.List;
+
+/**
+ * Created by Martin on 13.11.2016.
+ */
+@Service
+public interface VehicleService {
+    public Vehicle findById(Long id);
```
Comment (trebi) [inline fleet-management-service/src/main/java/cz/fi/muni/pa165/service/VehicleService.java]: **Look on this later:** Isn't this same as in `DAO`? Why do we need services?

**Pull request merged**